frontend/src/store/index.js

```diff
@@ -91,6 +241,24 @@ export default new Vuex.Store({
 		SET_AUTH_PROVIDERS(state, providers) {
 			state.providers = providers;
 		},
+		SET_BOARD(state, board) {
+			state.board = board;
+		},
+		SET_BOARDS(state, boards) {
+			state.boards = boards;
+		},
+		ADD_BOARD(state, board) {
+			state.boards.push(board);
+			//Vue.set(state.boards[state.boards.length], state.boards.length, board);
+		},
+		UPDATE_BOARD(state, board) {
+			const ix = state.boards.findIndex(b => b.id === board.id);
```

Comment:
If `ix == -1` it can cause problem. Better solution:
```js
const found = state.boards.find(b => b.id == board.id);
if (found) { Object.assign(found, board); }
else { state.boards.push(board); }
```
```